tasks/main.yml

```
@@ -67,6 +67,14 @@
         state: touch
       when: ansible_distribution == "Fedora"
 
+- name: "Set SELinux {{ clamav_seboolean_name }} flag persistently"
+  ansible.posix.seboolean:
+    name: "{{ clamav_seboolean_name }}"
+    persistent: yes
```

Prefer `true` to `yes` here as well:
```suggestion
    persistent: true
```